less/index.less

```diff
@@ -65,101 +70,142 @@ h1, h2, h3, h4, h5 {
 }
 
 .container {
-    width: 880px;
-    margin: auto;
+	width: 100%;
+	max-width: 800px + 80;
+  margin: auto;
+	background-color: @backgroundColor;
+	color: @fontColor;
+
+		@media (min-width:401px) and (max-width: 860px){
+			 padding: 0 percentage(50px / @container-width);
```

Comment:
looks good !~ media variable and math this one snippet demonstrates understanding of the topics for the day